pkg/internal/export/prom/prom.go

```diff
@@ -36,6 +37,8 @@ const (
 	rpcGRPCStatusCodeKey = "rpc_grpc_status_code"
 	rpcMethodKey         = "rpc_method"
 	rpcSystemGRPC        = "rpc_system"
+	DBOperationKey       = "db_operation"
+	DBSQLTableTable      = "db_sql_table"
```

Comment:
Similar here, we shouldn't put the DB table in the metrics for now. Let's just stick with operation, since it's guaranteed to be low cardinality if we parse things correctly.

I'll leave it up to you if you think we should put the table name, it can be under an option like we have for some of the HTTP attributes. If you choose to keep it with an option. it should be added as well for the OTel metrics (e.g. Prometheus and OTel should export the same things).

Honestly, I think we should leave it out for now and add it later if needed.